### PR TITLE
Use SCNu64 instead of ld when scanning for a 64-bit value

### PR DESCRIPTION
**Pull request by atiselsts**
This PR fixes the compilation of the current version on Raspberry Pi.

### CODE REVIEW

File: devtools/create-gossipstore.c [hunk 1]
```diff
@@ -28,7 +28,7 @@ struct scidsat * load_scid_file(FILE * scidfd)
 		return NULL;
 	struct scidsat * scids = calloc(n, sizeof(scidsat));
 	int i = 0;
-        while(fscanf(scidfd, "%s ,%ld\n", scids[i].scid, &scids[i].sat.satoshis) == 2 ) { /* Raw: read from file */
+        while(fscanf(scidfd, "%s ,%" SCNd64 "\n", scids[i].scid, &scids[i].sat.satoshis) == 2 ) { /* Raw: read from file */
```
Comment (cdecker) [inline devtools/create-gossipstore.c]: This should most likely be `SCNu64` (though it doesn't change much tbh).

**Comment by cdecker** [comment]
Thanks for the fix @atiselsts, I had been looking for something like this :-)

**Comment by atiselsts** [comment]
Updated to SCNu64 as requested!

**Comment by cdecker** [comment]
Excellent, thanks @atiselsts for the quick turnaround. I'll wait for CI to acknowledge that all is good and merge this :+1:

ACK https://github.com/ElementsProject/lightning/pull/2547/commits/3e3b41e3c8dd2729a17f1c0fb0525eeea152ac20

**Pull request merged**